CHANGELOG.rst

```diff
@@ -7,6 +7,48 @@ Synapse Changelog
 *****************
 
 
+v2.176.0 - 2024-07-18
+=====================
+
+Model Changes
+-------------
+- Updates to the ``inet`` model.
+  (`#3811 <https://github.com/vertexproject/synapse/pull/3811>`_)
+  (`#3814 <https://github.com/vertexproject/synapse/pull/3814>`_)
+
+  **New Forms**
+
+  ``inet:service:thread``
+    A message thread.
+
+  **New Properties**
+
+  ``inet:service:message``
+    The form had the following properties added to it:
+
+    ``thread``
+      The thread which contains the message.
+
+    ``title``
+      The message title.
+
+  **Updated Forms**
+
+  ``inet:service:account``
+    The form now inherits from the ``inet:service:object`` interface.
+
+Features and Enhancements
+-------------------------
+- Add ``storm.exec`` command for executing arbitrary text as Storm.
+  (`#3807 <https://github.com/vertexproject/synapse/pull/3807>`_)
+  (`#3812 <https://github.com/vertexproject/synapse/pull/3812>`_)
```

Comment:
Looks like this PR just changed formatting for the changelog entry, so should it be listed here?

Reply:
Fixed formatting + fixed code around the original PR too ;)